demisto_sdk/commands/common/constants.py

```diff
@@ -1845,5 +1845,7 @@ class ParameterType(Enum):
 FORMATTING_SCRIPT = "indicator-format"
 
 ENV_SDK_WORKING_OFFLINE = "DEMISTO_SDK_OFFLINE_ENV"
+DOCKERFILES_INFO_REPO = "demisto/dockerfiles-info"
+PYTHON_IMAGE_REGEX = re.compile(r"[\d\w]+/python3?:(?P<python_version>[23]\.\d+)")
```

Comment:
won't the python3 in the first part of the regex prevent this from working? why only in the second part you're doing python[23]

Reply:
its the same regex like in https://github.com/demisto/demisto-sdk/pull/3339/files#diff-6f1f6f0276b6ce9340907841f027175e315cf55853561f31c0e2920febba3674L39,

i just copied it from docker_helper to constants, however i will revert it completely